dtfs-central-api/src/helpers/amendments/tfm-facility-latest-completed-amendment-to-facility-value.ts

```diff
@@ -0,0 +1,25 @@
+import { TfmFacility } from '@ukef/dtfs2-common';
+import orderBy from 'lodash.orderby';
+import { filterCompletedAmendments } from './filter-completed-amendments';
+
+/**
+ * Get the latest completed facility value amendment
+ * @param tfmFacility - The tfm facility
+ * @returns The latest completed facility value amendment or null if there aren't any
+ */
+export const getLatestCompletedAmendmentToFacilityValue = (tfmFacility: TfmFacility): number | null => {
+  const { amendments } = tfmFacility;
+  if (!amendments) {
+    return null;
+  }
+
+  const completedAmendments = filterCompletedAmendments(amendments);
+
+  const latestAmendmentWithFacilityValue = orderBy(completedAmendments, ['updatedAt'], ['desc']).find(({ value }) => value !== undefined && value !== null);
+
+  if (latestAmendmentWithFacilityValue?.value !== undefined && latestAmendmentWithFacilityValue?.value !== null) {
```

Comment:
Could this just be `if(latestAmendmentWithFacilityValue) {` since we check for undefined and null in the previous line?

Reply:
I changed this because I realised it is a number so wanted to be explicit to avoid potentially handling zero incorrectly